### PR TITLE
Limpbrains jest27 wip

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ke5aztcenqa9082cupb14uwm04?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10004)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2pw2am67dgpzqb9h0ncz9c76ym?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10005)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2qnf8vtejgjxz15bb7nu84vbug?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10006)

**Review by Mack2208 — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/uvgkemdpuwycqeqw0yybxnvvtw?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10008)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wq83y47gbfvq60h15n3nfyy970?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10009)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2m8u864pq9x0xk6njg57kwdz3m?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10015)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/11fgmrhmyk00whryckwg2ppd4m?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10016)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dd623h6v5jctgwr06jejy95a0r?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10017)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4h3v21f14c5rbymt7u7u40exf4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10019)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/grzqdmd6tq3gxrcmp8h8w4tjxm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10020)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/yz6pzacm756f061j38xvq69qk0?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10022)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/5j0pft0f932rr5eaveum43hkm4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10023)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xqccwzjewwt5pvbfdxvkxcvwjw?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10024)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/va7w52hzx52tqenpwdg5jwjk9c?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10025)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/j0dybw56z5xqb465zh4gz7gtjw?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10026)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/gctw823rrz6y5x02cnnpjprcqr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10027)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/7p3f8fvhv0jcgtddx30y9h3ddw?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10028)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/19gj716r2wd03y3fab5t0gpbbm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10029)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/71cfja47tymj4ugwpq3jzb86jr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10030)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6td60wh16p16bm0vf8dhteqa68?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10032)

File: package.json [hunk 1]
```diff
@@ -125,7 +125,7 @@
     "coinselect": "3.1.12",
     "crypto-js": "4.1.1",
     "dayjs": "1.11.2",
-    "detox": "19.4.4",
+    "detox": "^19.6.9",
```
Comment (limpbrains) [inline package.json:128]: Need to lock this version

File: tests/unit/hd-aezeed.test.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { HDAezeedWallet, WatchOnlyWallet } from '../../class';
 const assert = require('assert');
 
 describe('HDAezeedWallet', () => {
-  it('can import mnemonics and generate addresses and WIFs', async function () {
+  it.only('can import mnemonics and generate addresses and WIFs', async function () {
```
Comment (limpbrains) [inline tests/unit/hd-aezeed.test.js:5]: .only

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: tests/unit/hd-aezeed.test.js (hunk 2 above)
Comment (Overtorment) [reply tests/unit/hd-aezeed.test.js:5]: good catch

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1nr0gzpxcqvj3hz7u63ytpv68c?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10034)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rawph3h7y7rrgkvummhq5cue1m?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10035)

File: .circleci/config.yml [hunk 3]
```diff
@@ -20,4 +21,63 @@ jobs:
...
+
+      - save_cache:
+          key: node_modules-{{ checksum "package-lock.json" }}
+          paths:
+            - node_modules
+
+      # run tests!
+      - run:
+          command: npm run jest || npm run jest || npm run jest
+          no_output_timeout: 60m
```
Comment (limpbrains) [inline .circleci/config.yml:75]: you probably want to remove this

File: package.json (hunk 1 above)
Comment (limpbrains) [reply package.json:128]: and this

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/af83knagyw9j711uayjrqfawqr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10037)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vpzdgq5cc2k3r5uryjjpqzpgp0?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10038)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mcgge6vu2tv4htzbb25tp6er94?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10039)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/unqjhm5rja39tyx3jcjvycffrr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10043)

File: .circleci/config.yml (hunk 3 above)
Comment (limpbrains) [reply .circleci/config.yml:75]: Yes

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pm78v5uyq65ddz3588u9p17hfr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10061)

**Review by limpbrains — APPROVED** [review]
Awesome, now we can start migrating to jest 28!

**Comment by Overtorment** [comment]
> Awesome, now we can start migrating to jest 28!

😲

**Pull request merged**